test/unit/managers/test_CollectionManager.py

```diff
@@ -100,14 +100,16 @@ def test_update_from_dict(self):
             'deleted': True,
             'visible': False,
             'name': 'New Name',
+            'tags': ['one', 'two', 'three']
             # TODO: doesn't work
-            # 'tags'      : [ 'one', 'two', 'three' ]
             # 'annotations'      : [?]
         })
         self.assertEqual(hdca.name, 'New Name')
         self.assertTrue(hdca.deleted)
         self.assertFalse(hdca.visible)
-        # self.assertEqual( hdca.tags, [ 'one', 'two', 'three' ] )
+
+        tag_names = [t.tag.name for t in hdca.tags]
```

Comment:
```suggestion
        tag_names = [t.to_string() for t in hdca.tags]
```
if you're going for the `to_string` method